crates/pevm/src/storage/rpc.rs

```diff
@@ -37,8 +44,7 @@ pub struct RpcStorage<N: Network> {
     cache_accounts: Mutex<ChainState>,
     cache_bytecodes: Mutex<Bytecodes>,
     cache_block_hashes: Mutex<BlockHashes>,
-    // TODO: Better async handling.
-    runtime: Runtime,
+    runtime: OnceLock<Runtime>,
```

Comment:
Very nit but we can move this to the top now to let the caches flow to the bottom (now that there's no comment to separate them and the runtime).